helper/call.go

```diff
@@ -474,6 +498,25 @@ func (c *ContractCaller) EncodeStateSyncedEvent(log *ethTypes.Log) (*statesender
 	return event, nil
 }
 
+// FindStakedEventLogIndex find log index for staked event
+func (c *ContractCaller) FindStakedEventLogIndex(receipt *ethTypes.Receipt) (uint64, bool) {
+	abiObject := &c.StakingInfoABI
+	eventName := "Staked"
```

Comment:
Probably add this string as a const and use in both `FindStakedEventLogIndex` and `DecodeValidatorJoinEvent`, I think we have a list of constants in some package.

Reply:
We will remove `FindStakedEventLogIndex` method in cs-2003 and only use `DecodeValidatorJoinEvent` using `msg.LogIndex`.